### DISCUSSION TITLE
el-table summary-method方法如何添加两行合计行

### DISCUSSION THREAD

**Discussion by wasonChen** [element-plus/element-plus#16569] [Q&A]
官方的summary-method方法是默认一行合计行,但是我的需求是要实现两行的合计行,要如何实现呢?

**Comment by jinceon** [comment]
合并行或列[#](https://element-plus.org/zh-CN/component/table.html#%E5%90%88%E5%B9%B6%E8%A1%8C%E6%88%96%E5%88%97)
多行或多列共用一个数据时，可以合并行或列。

通过给 table 传入span-method方法可以实现合并行或列


自己实现。
